astropy/nddata/mixins/ndarithmetic.py

```diff
@@ -8,14 +8,138 @@
 
 import numpy as np
 
-from ... units import dimensionless_unscaled, UnitsError
-from ... import log
-from ...extern.six import string_types
-from ..nduncertainty import IncompatibleUncertaintiesException
+from ..nduncertainty import NDUncertainty
+from ...units import dimensionless_unscaled
+from ...extern import six
 
 __all__ = ['NDArithmeticMixin']
 
 
+# TODO: General question of implementing an Operator class #3861? (embray)
+# Maybe a bit too big for these changes.
+
+try:
+    from .utils import format_doc
+except ImportError:
+    # TODO: Delete this and rebase if #4242 is merged.
+    def format_doc(docstring, *args, **kwargs):
```

Comment:
Looks like #4242 is close to done...so I wouldn't remove and rebase this *today* but once #4242 is merged go ahead.